src/api/theMovieDB.js

```diff
@@ -0,0 +1,15 @@
+const MOVIE_API_BASE_URL = `https://api.themoviedb.org/3/discover/movie?api_key=${process.env.MOVIE_API_KEY}`;
+
+/**
+ * Fetch movie's data from themoviedb api
+ *
+ * @param {number} page - Number of the page to query
+ * @returns {object}
+ */
+async function getMovies(page) {
+  const respnse = await fetch(`${MOVIE_API_BASE_URL}&page=${page}`);
+
+  return await respnse.json();
```

Comment:
You don't need to add `await` after the `return` keyword.

An `async` function returns a `Promise` that resolves to the value returned from the function and `await` waits for the `Promise` to be resolved before continuing the execution of an `async` function. If an `async` function returns a `Promise` as the return value then you don’t need to use await after the return keyword.

and lucky you `respnse.json()` returns a `Promise` 🍻 

btw you have a type in `respnse` should be `response`

Reply:
Nice catch!!! Thanks!!